components/perl/perl-534/Makefile

```diff
@@ -0,0 +1,110 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright (c) 2013, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2021, Tim Mooney.  All rights reserved.
+#
+BUILD_STYLE=configure
+BUILD_BITS=64
+include ../../../make-rules/shared-macros.mk
+
+#
+# Note this perl is 64-bit only.
+#
+
+# Override PERL_VERSION because when building a new version
+# of perl it probably isn't installed so shared-macros.mk
+# can't call it.
+
+COMPONENT_NAME=         perl
+COMPONENT_VERSION=      5.34.0
+COMPONENT_SUMMARY=      Perl 5.30
```

Comment:
5.34?

Reply:
Nice catch @pyhalov !  I'll fix that tomorrow when I have some focused work time.

It's a leftover from when I originally started the perl update, just before the pandemic started.  I was originally building 5.30.0.

Nice to see you're still checking in on OI once in a while.  I hope things are great with you!

Reply:
If perl architecture names do change with this new perl version, please also verify their consitency in:
oi-userland/tools/python/pkglint/userland.py
Thanks